lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -418,7 +456,7 @@ def main():
                     current_version = response['Version']
                 changed = True
             except (botocore.exceptions.ParamValidationError, botocore.exceptions.ClientError) as e:
-                module.fail_json(msg=str(e))
+                fail_json_aws(module, e, msg="updating function code")
```

Comment:
We should never even get ParamValidationErrors, that's a sign we haven't done the right thing somewhere.